tests/CMakeLists.txt

```diff
@@ -21,6 +21,10 @@ if(UNIX)
     set_target_properties(vk_loader_validation_tests PROPERTIES COMPILE_FLAGS "-Wno-sign-compare")
 endif()
 
+if(WIN32)
```

Comment:
This if(WIN32) could be folded into the if(WIN32) immediately following.

Reply:
Fixed.